dev/WindowsAppRuntime_BootstrapDLL/packages.config

```diff
@@ -1,4 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is there a way to at least centralize these `packages.config` files across the solution?

Reply:
Yes but not easily (VS doesn't provide a mechanism but we can, with work). It's come up in ES discussions. Ideas so far include...

* build\packages.props as a single shared master list of WIL, C++/WinRT, etc definitions/versions instead of in each *proj file
* new tooling (UpdatePackagesConfig.ps1 etc) to crawl the directory tree rewriting packages.config content per packages.props
* pipeline checks all packages.config match (and match versions in packages.props)
* Non-nuget packaging tech (eg VCPKG)

This overlaps the project-management topic we've discussed but haven't tackled yet. But that day's getting closer... :-)